# Make possible for CFLAGS from environment, remove $(LIBS) from dependencies and do not hide commands
diff -urNad MiniDLNAorig//Makefile minidlna/Makefile
--- MiniDLNAorig//Makefile	2010-07-17 00:25:54.000000000 +0300
+++ minidlna/Makefile	2017-04-22 13:33:59.386627768 +0300
@@ -12,7 +12,7 @@
 #
 #CFLAGS = -Wall -O -D_GNU_SOURCE -g -DDEBUG
 #CFLAGS = -Wall -g -Os -D_GNU_SOURCE
-CFLAGS = -Wall -g -O3 -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
+CFLAGS += -Wall -g -O3 -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
 	 -I/usr/include/ffmpeg \
 	 -I/usr/include/libavutil -I/usr/include/libavcodec -I/usr/include/libavformat \
 	 -I/usr/include/ffmpeg/libavutil -I/usr/include/ffmpeg/libavcodec -I/usr/include/ffmpeg/libavformat
@@ -58,14 +58,14 @@
 	$(INSTALL) -d $(ETCINSTALLDIR)
 	$(INSTALL) --mode=0644 minidlna.conf $(ETCINSTALLDIR)
 
-minidlna:	$(BASEOBJS) $(LNXOBJS) $(LIBS)
+minidlna:	$(BASEOBJS) $(LNXOBJS)
 	@echo Linking $@
-	@$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(BASEOBJS) $(LNXOBJS) $(LIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(BASEOBJS) $(LNXOBJS) $(LIBS)
 
 
 testupnpdescgen:	$(TESTUPNPDESCGENOBJS)
 	@echo Linking $@
-	@$(CC) $(CFLAGS) -o $@ $(TESTUPNPDESCGENOBJS)
+	$(CC) $(CFLAGS) -o $@ $(TESTUPNPDESCGENOBJS)
 
 config.h:	genconfig.sh
 	./genconfig.sh
@@ -123,7 +123,7 @@
 
 .c.o:
 	@echo Compiling $*.c
-	@$(CC) $(CFLAGS) -o $@ -c $< && exit 0;\
+	$(CC) $(CFLAGS) -o $@ -c $< && exit 0;\
 		echo "The following command failed:" 1>&2;\
 		echo "$(CC) $(CFLAGS) -o $@ -c $<";\
 		$(CC) $(CFLAGS) -o $@ -c $< &>/dev/null
